reference/kubectl-plugin.md

```diff
@@ -144,7 +149,7 @@ Volume 0c08667c-8b59-4d11-9192-b54e27e0ce0f Scaled Successfully 🚀
 {% endtabs %}
 
 
-- Retrieving resource specs in any desired format
+### Retrieve resource in any output format
```

Comment:
Retrieve resource details in the output formats table, json or yaml, with table being the default output format